requirements.txt

```diff
@@ -79,7 +79,7 @@ flask-talisman==0.8.1
     # via -r requirements.in
 flask-wtf==1.0.0
     # via -r requirements.in
-funding-service-design-utils @ https://github.com/communitiesuk/funding-service-design-utils/archive/refs/tags/0.0.20.tar.gz
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils.git@fs-1450-is-authenticated-flag
```

Comment:
Can we use a released version before this is merged?

Reply:
Yep will do - obv this can't work on dev without using the right branch that's all

Reply:
Updated to use 0.0.21